lib/renuo/bin-check/master_thread.rb

```diff
@@ -21,8 +22,20 @@ def finalize
       until waiter.empty?
         thread = waiter.next_wait
         @results << thread[:result]
-        exit 1 if @results.last.exit_code == 1
+        exit_with_error if @results.last.exit_code == 1
```

Comment:
don't we also want to give developer feedback asap? then we may also want to print if a check ran successfully?